lineman/app/components/thread_page/revision_history_modal/revision_history_modal.haml

```diff
@@ -0,0 +1,18 @@
+.revision-history-modal
```

Comment:
This will involve a little work to get into the new build system that's in master now; namely:

- Will need to copy component files from lineman/app to angular/core
- Need to update the haml, as the parser we're using has changed slightly. Biggest change is that all of the single quotes have to become double quotes for the parser to handle them correctly.